### PR TITLE
update serverless connect steps

### PR DESCRIPTION
**Pull request by zhangyangyu**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/secure-connections-to-serverless-clusters.md [hunk 1]
```diff
@@ -183,8 +85,8 @@ In different operating systems, the default storage paths of the root certificat
 
 Windows does not offer a specific path to the CA root. Instead, it uses the [registry](https://learn.microsoft.com/en-us/windows-hardware/drivers/install/local-machine-and-current-user-certificate-stores) to store certificates. For this reason, to specify the CA root path on Windows, take the following steps:
 
-1. Download the [Mozilla CA Certificate bundle](https://curl.se/docs/caextract.html) and save it in a path you prefer, such as `<path_to_mozilla_ca_cert_bundle>`.
-2. Use the path (`<path_to_mozilla_ca_cert_bundle>`) as your CA root path when you connect to a TiDB Serverless cluster.
+1. Download the [ISRG Root X1](https://letsencrypt.org/certs/isrgrootx1.pem.txt) certificate. and save it in a path you prefer, such as `<path_to_ca>`.
```
Comment (hfxsd) [inline tidb-cloud/secure-connections-to-serverless-clusters.md:88]:
```suggestion
1. Download the [ISRG Root X1 certificate](https://letsencrypt.org/certs/isrgrootx1.pem.txt) and then save it in a path you prefer, such as `<path_to_ca>`.
```

File: tidb-cloud/secure-connections-to-serverless-clusters.md [hunk 2]
```diff
@@ -40,108 +37,13 @@ In the [TiDB Cloud console](https://tidbcloud.com/), you can get examples of dif
...
-
-However, some drivers and ORMs do not use the system root CA stores. In those cases, you need to configure the CA root path of the drivers or ORMs to your system root CA stores. For example, when you use [mysqlclient](https://github.com/PyMySQL/mysqlclient) to connect a TiDB Serverless cluster in Python on macOS, you need to set `ca: /etc/ssl/cert.pem` in the `ssl` argument.
+If the client uses the system's root CA stores by default, such as Java and Go, you can easily connect securely to TiDB Serverless clusters without specifying the path of CA roots. However, some drivers and ORMs do not use the system root CA stores. In those cases, you need to configure the CA root path of the drivers or ORMs to your system root CA stores. For example, when you use [mysqlclient](https://github.com/PyMySQL/mysqlclient) to connect a TiDB Serverless cluster in Python on macOS, you need to set `ca: /etc/ssl/cert.pem` in the `ssl` argument.
```
[the remark this answers is not in the document]
Comment (zhangyangyu) [reply tidb-cloud/secure-connections-to-serverless-clusters.md:46]: 完全不需要，用户在连接串上根本就看不见 CA 信息。

File: tidb-cloud/branch-manage.md [hunk 3]
```diff
@@ -46,13 +46,9 @@ To connect to a branch, perform the following steps:
 2. Click **Branches** in the left navigation pane.
 3. In the row of your target branch to be connected, click **...** in the **Action** column.
 4. Click **Connect** in the drop-down list. The dialog for the connection information is displayed.
-5. Click **Create password** or **Reset password** to create or reset the root password.
+5. Click **Generate Password** or **Reset Password** to create or reset the root password.
 6. Connect to the branch using the connection information.
```
Comment (qiancai) [inline tidb-cloud/branch-manage.md:50]: L41  Connect to a branch 这里要不要也提供如何从 Overview 页面右上角的 Connect 来连接

Comment (zhangyangyu) [reply tidb-cloud/branch-manage.md:50]: 加上了

**Review by shiyuhang0 — APPROVED** [review]

File: tidb-cloud/connect-via-standard-connection-serverless.md [hunk 4]
```diff
@@ -24,9 +24,11 @@ To connect to a TiDB Serverless cluster via public endpoint, take the following
     > - Keeping the endpoint type as `Public` means the connection is via standard TLS connection. For more information, see [TLS Connection to TiDB Serverless](/tidb-cloud/secure-connections-to-serverless-clusters.md).
     > - If you choose **Private** in the **Endpoint Type** drop-down list, it means that the connection is via private endpoint. For more information, see [Connect to TiDB Serverless via Private Endpoint](/tidb-cloud/set-up-private-endpoint-connections-serverless.md).
 
-4. If you have not set a password yet, click **Create password** to generate a random password. The generated password will not show again, so save your password in a secure location.
+4. TiDB Serverless lets you create [branches](/tidb-cloud/branch-overview.md). You can choose to connect to each branch via **Branch** drop-down list. `main` represents the cluster itself.
```
Comment (hfxsd) [inline tidb-cloud/connect-via-standard-connection-serverless.md:27]:
```suggestion
4. TiDB Serverless lets you create [branches](/tidb-cloud/branch-overview.md). You can choose to connect to a branch via the **Branch** drop-down list. `main` represents the cluster itself.
```

File: tidb-cloud/branch-manage.md [hunk 5]
```diff
@@ -46,12 +46,16 @@ To connect to a branch, perform the following steps:
...
 
-> **Note:**
->
-> Currently, branches do not support [private endpoints](/tidb-cloud/set-up-private-endpoint-connections-serverless.md).
+Or you can get the connection string from cluster overview page:
+
+1. In the [TiDB Cloud console](https://tidbcloud.com/), navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page of your project, and then click the name of your target TiDB Serverless cluster to go to its overview page.
+2. Click **Connect** in the upper-right corner.
+3. Select the branch you want to connect in the `Branch` drop-down list.
```
Comment (hfxsd) [inline tidb-cloud/branch-manage.md:56]:
```suggestion
3. Select the branch you want to connect to in the `Branch` drop-down list.
```

File: tidb-cloud/branch-manage.md [hunk 6]
```diff
@@ -46,12 +46,16 @@ To connect to a branch, perform the following steps:
...
 3. In the row of your target branch to be connected, click **...** in the **Action** column.
 4. Click **Connect** in the drop-down list. The dialog for the connection information is displayed.
-5. Click **Create password** or **Reset password** to create or reset the root password.
+5. Click **Generate Password** or **Reset Password** to create or reset the root password.
 6. Connect to the branch using the connection information.
 
-> **Note:**
->
-> Currently, branches do not support [private endpoints](/tidb-cloud/set-up-private-endpoint-connections-serverless.md).
+Or you can get the connection string from cluster overview page:
```
Comment (hfxsd) [inline tidb-cloud/branch-manage.md:52]:
```suggestion
Alternatively, you can get the connection string from cluster overview page:
```

File: develop/dev-guide-sample-application-golang-gorm.md [hunk 7]
```diff
@@ -76,11 +76,11 @@ Connect to your TiDB cluster depending on the TiDB deployment option you've sele
     >
```
Comment (hfxsd) [inline develop/dev-guide-sample-application-golang-gorm.md:76]: L72 的 `General` 需要改成 `Go` 吗

File: develop/dev-guide-sample-application-golang-sql-driver.md [hunk 8]
```diff
@@ -76,11 +76,11 @@ Connect to your TiDB cluster depending on the TiDB deployment option you've sele
     >
```
Comment (hfxsd) [inline develop/dev-guide-sample-application-golang-sql-driver.md:76]: L72 的 `General` 需要改成 `Go` 吗

File: develop/dev-guide-sample-application-java-hibernate.md [hunk 9]
```diff
@@ -77,11 +77,11 @@ Connect to your TiDB cluster depending on the TiDB deployment option you've sele
     >
```
Comment (hfxsd) [inline develop/dev-guide-sample-application-java-hibernate.md:77]: L73 的 `General` 是否也需要更新

File: develop/dev-guide-sample-application-java-jdbc.md [hunk 10]
```diff
@@ -77,11 +77,11 @@ Connect to your TiDB cluster depending on the TiDB deployment option you've sele
     >
     > If your program is running in Windows Subsystem for Linux (WSL), switch to the corresponding Linux distribution.
 
-4. Click **Create password** to create a random password.
+4. Click **Generate Password** to create a random password.
```
Comment (hfxsd) [inline develop/dev-guide-sample-application-java-jdbc.md:81]: L73 的 `General` 是否也需要更新

File: develop/dev-guide-sample-application-nodejs-mysql2.md [hunk 11]
```diff
@@ -89,7 +89,7 @@ Connect to your TiDB cluster depending on the TiDB deployment option you've sele
     - **Connect With** is set to `General`.
```
Comment (hfxsd) [inline develop/dev-guide-sample-application-nodejs-mysql2.md:89]:
```suggestion
    - **Connect With** is set to `Node.js(mysql2)`.
```

File: develop/dev-guide-sample-application-python-django.md [hunk 12]
```diff
@@ -95,11 +95,11 @@ Connect to your TiDB cluster depending on the TiDB deployment option you've sele
     >
```
Comment (hfxsd) [inline develop/dev-guide-sample-application-python-django.md:95]: L91 的 `General` 是否要改成 `Django`

File: develop/dev-guide-gui-datagrip.md [hunk 13]
```diff
@@ -56,14 +56,14 @@ Connect to your TiDB cluster depending on the TiDB deployment option you've sele
 3. Ensure the configurations in the connection dialog match your operating environment.
 
     - **Endpoint Type** is set to `Public`
-    - **Connect With** is set to `JDBC`
+    - **Connect With** is set to `DataGrip`
```
Comment (hfxsd) [inline develop/dev-guide-gui-datagrip.md:59]: `Branch` 的步骤也需要加上吗
```suggestion
    - **Branch** is set to `main`
    - **Connect With** is set to `DataGrip`
```

File: develop/dev-guide-sample-application-nodejs-prisma.md [hunk 14]
```diff
@@ -86,21 +86,22 @@ Connect to your TiDB cluster depending on the TiDB deployment option you've sele
...
+4. If you have not set a password yet, click **Generate Password** to generate a random password.
 
 5. Run the following command to copy `.env.example` and rename it to `.env`:
 
     ```shell
     cp .env.example .env
     ```
 
-6. Edit the `.env` file, set up the environment variable `DATABASE_URL` as follows, replace the corresponding placeholders `{}` with connection parameters on the connection dialog:
+6. Edit the `.env` file, set up the environment variable `DATABASE_URL` as follows, replace the corresponding placeholders `{}` with connection string on the connection dialog:
```
Comment (qiancai) [inline develop/dev-guide-sample-application-nodejs-prisma.md:101]:
```suggestion
6. Edit the `.env` file, set up the environment variable `DATABASE_URL` as follows, replace the corresponding placeholders `{}` with connection string on the connection dialog:
```
```suggestion
6. Edit the `.env` file, set up the environment variable `DATABASE_URL` as follows, and replace the corresponding placeholders `{}` with the connection string in the connection dialog:
```

File: develop/dev-guide-sample-application-python-mysql-connector.md [hunk 15]
```diff
@@ -77,18 +77,19 @@ Connect to your TiDB cluster depending on the TiDB deployment option you've sele
 3. Ensure the configurations in the connection dialog match your operating environment.
 
     - **Endpoint Type** is set to `Public`
+    - **Branch** is set to `main`
     - **Connect With** is set to `General`
```
Comment (qiancai) [inline develop/dev-guide-sample-application-python-mysql-connector.md:81]: **Connect With** 要选 Python (MySQL Connector) 吗

File: develop/dev-guide-sample-application-python-mysqlclient.md [hunk 16]
```diff
@@ -79,18 +79,19 @@ Connect to your TiDB cluster depending on the TiDB deployment option you've sele
 3. Ensure the configurations in the connection dialog match your operating environment.
 
     - **Endpoint Type** is set to `Public`
+    - **Branch** is set to `main`
     - **Connect With** is set to `General`
```
Comment (qiancai) [inline develop/dev-guide-sample-application-python-mysqlclient.md:83]: Connect With 要选 Python (mysqlclient) 吗

File: develop/dev-guide-sample-application-python-pymysql.md [hunk 17]
```diff
@@ -77,18 +77,19 @@ Connect to your TiDB cluster depending on the TiDB deployment option you've sele
 3. Ensure the configurations in the connection dialog match your operating environment.
 
     - **Endpoint Type** is set to `Public`
+    - **Branch** is set to `main`
     - **Connect With** is set to `General`
```
Comment (qiancai) [inline develop/dev-guide-sample-application-python-pymysql.md:81]: Connect With 要选 Python (PyMySQL) 吗

File: tidb-cloud/branch-manage.md [hunk 18]
```diff
@@ -46,12 +46,16 @@ To connect to a branch, perform the following steps:
...
 3. In the row of your target branch to be connected, click **...** in the **Action** column.
 4. Click **Connect** in the drop-down list. The dialog for the connection information is displayed.
-5. Click **Create password** or **Reset password** to create or reset the root password.
+5. Click **Generate Password** or **Reset Password** to create or reset the root password.
 6. Connect to the branch using the connection information.
 
-> **Note:**
->
-> Currently, branches do not support [private endpoints](/tidb-cloud/set-up-private-endpoint-connections-serverless.md).
+Alternatively, you can get the connection string from cluster overview page:
```
Comment (qiancai) [inline tidb-cloud/branch-manage.md:52]:
```suggestion
Alternatively, you can get the connection string from the cluster overview page:
```

File: tidb-cloud/branch-manage.md [hunk 19]
```diff
@@ -46,12 +46,16 @@ To connect to a branch, perform the following steps:
...
+
+1. In the [TiDB Cloud console](https://tidbcloud.com/), navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page of your project, and then click the name of your target TiDB Serverless cluster to go to its overview page.
+2. Click **Connect** in the upper-right corner.
+3. Select the branch you want to connect to in the `Branch` drop-down list.
+4. Click **Generate Password** or **Reset Password** to create or reset the root password.
+5. Connect to the branch using the connection information.
```
Comment (qiancai) [inline tidb-cloud/branch-manage.md:59]: 确认下，原来的这个 note 限制现在不需要了吗

File: tidb-cloud/connect-via-standard-connection-serverless.md [hunk 20]
```diff
@@ -24,9 +24,11 @@ To connect to a TiDB Serverless cluster via public endpoint, take the following
     > - Keeping the endpoint type as `Public` means the connection is via standard TLS connection. For more information, see [TLS Connection to TiDB Serverless](/tidb-cloud/secure-connections-to-serverless-clusters.md).
     > - If you choose **Private** in the **Endpoint Type** drop-down list, it means that the connection is via private endpoint. For more information, see [Connect to TiDB Serverless via Private Endpoint](/tidb-cloud/set-up-private-endpoint-connections-serverless.md).
 
-4. If you have not set a password yet, click **Create password** to generate a random password. The generated password will not show again, so save your password in a secure location.
+4. TiDB Serverless lets you create [branches](/tidb-cloud/branch-overview.md). You can choose to connect to a branch via the **Branch** drop-down list. `main` represents the cluster itself.
```
Comment (qiancai) [inline tidb-cloud/connect-via-standard-connection-serverless.md:27]:
```suggestion
4. TiDB Serverless lets you create [branches](/tidb-cloud/branch-overview.md) for your cluster. After a branch is created, you can choose to connect to the branch via the **Branch** drop-down list. `main` represents the cluster itself.
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

File: develop/dev-guide-sample-application-python-mysqlclient.md (hunk 16 above)
Comment (zhangyangyu) [reply develop/dev-guide-sample-application-python-mysqlclient.md:83]: 同样不需要

File: develop/dev-guide-sample-application-python-pymysql.md (hunk 17 above)
Comment (zhangyangyu) [reply develop/dev-guide-sample-application-python-pymysql.md:81]: 同样不需要

File: tidb-cloud/branch-manage.md (hunk 19 above)
Comment (zhangyangyu) [reply tidb-cloud/branch-manage.md:59]: 是的，branch 我们也开放了 private link 功能，可以在 connect 上选择

**Pull request merged**